app/src/routes/v1/ipc.js

```diff
@@ -51,7 +51,7 @@ router.get('/:ipcPlanId', keycloak.protect(), async (req, res, next) => {
   }
 });
 
-router.get('/pdf/:ipcPlanId', keycloak.protect(), async (req, res, next) => {
+router.get('/pdf/:ipcPlanId', async (req, res, next) => {
```

Comment:
Discuss after getting this working with Jason if we want to rate limit this endpoint now